kroxylicious-filters/kroxylicious-record-encryption/src/main/java/io/kroxylicious/filter/encryption/RecordEncryptionFilter.java

```diff
@@ -139,6 +150,17 @@ private CompletionStage<ProduceRequestData> maybeEncodeProduce(ProduceRequestDat
                 });
     }
 
+    private void generateMetrics(String virtualClusterName, Set<String> unresolvedTopicNames, Map<String, TopicProduceData> topicNameToData) {
+        unresolvedTopicNames.forEach(unresolvedTopic -> {
+            var unEncyptedRecordsTotal = RecordEncryptionMetrics.recordEncryptionPlainRecordsCounter(virtualClusterName, unresolvedTopic).withTags();
```

Comment:
consistency would be nice here. The records are either `plain` or `unencrypted` but both just begs the question "whats the difference between plain and unencrypted?"